SWLOR.Game.Server/Feature/RecipeDefinition/EngineeringRecipeDefinition/ModuleRecipes.cs

```diff
@@ -264,7 +264,7 @@ private void Tier1()
                 .Component("ref_tilarium", 3)
                 .Component("elec_ruined", 2);
 
-            // Interceptor Config 1
+            // Basic Interceptor Config
             _builder.Create(RecipeType.InterceptorConfig1, SkillType.Engineering)
```

Comment:
If you're going to rename the items, can you make sure the enumerations also match? It's difficult to change it after this releases so I'd like to make the correction now if at all possible.